src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java

```diff
@@ -450,14 +463,29 @@ public String getQuestionResultStatisticsCsv(
 
         DecimalFormat df = new DecimalFormat("#.##");
         
+        Map<String, List<Integer>> sortedOptionPoints = new TreeMap<String, List<Integer>>();
+        
+        Map<String, String> identifierMap = new HashMap<String, String>();
+        
         for (Entry<String, List<Integer>> entry : optionPoints.entrySet()) {
+            
+            if (distributeToRecipients) {
+                sortForRecipient(entry, identifierMap, sortedOptionPoints, bundle);
```

Comment:
Since `optionPoints` contains either recipient mappings or option mappings, we could pass `optionPoints` directly into the method (instead of `entry`), and bring the `for` loop into the method. That way the boolean condition is only checked once before sorting and not on every iteration of the loop. Although it was done this way previously, let's fix it here 😛